ngraph/core/include/openvino/op/nv12_to_rgb.hpp

```diff
@@ -0,0 +1,63 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "openvino/op/util/convert_color_nv12_base.hpp"
+
+namespace ov {
+namespace op {
+namespace v8 {
+/// \brief Color conversion operation from NV12 to RGB format.
+///    Input:
+///        - Input NV12 image can be represented in two ways:
+///            a) Single plane (as it is in the file): NV12 height dimension is 1.5x bigger than image height. 'C'
+///               dimension shall be 1.
+///            b) Two separate planes (used this way in many physical video sources): Y and UV. In
+///               this case
+///               b1) Y plane has height same as image height. 'C' dimension equals to 1
+///               b2) UV plane has dimensions: 'H' = image_h / 2; 'W' = image_w / 2; 'C' = 2.
+///        - Supported element types: u8 or any supported floating-point type.
+///    Output:
+///        - Output node will have NHWC layout and shape HxW same as image spatial dimensions.
+///        - Number of output channels 'C' will be 3, as per interleaved RGB format, first channel is R, last is B
+///
+/// \details Conversion of each pixel from NV12 (YUV) to RGB space is represented by following formulas:
+///        R = 1.164 * (Y - 16) + 1.596 * (V - 128)
+///        G = 1.164 * (Y - 16) - 0.813 * (V - 128) - 0.391 * (U - 128)
+///        B = 1.164 * (Y - 16) + 2.018 * (U - 128)
+///        Then R, G, B values are clipped to range (0, 255)
+///
+class OPENVINO_API NV12toRGB : public util::ConvertColorNV12Base {
+public:
+    OPENVINO_OP("NV12toRGB", "opset8", util::ConvertColorNV12Base);
+    BWDCMP_RTTI_DECLARATION;
```

Comment:
We don't need this macro if the base class also doesn't define it.

Reply:
fixed